src/domain/community/organization/organization.resolver.queries.ts

```diff
@@ -13,8 +13,28 @@ export class OrganizationResolverQueries {
     description: 'The Organizations on this platform',
   })
   @Profiling.api
-  async organizations(): Promise<IOrganization[]> {
-    return await this.organizationService.getOrganizations();
+  async organizations(
+    @Args({
+      name: 'limit',
+      type: () => Float,
+      description:
+        'The number of Organizations to return; if omitted return all Organizations.',
+      nullable: true,
+    })
+    limit: number,
+    @Args({
+      name: 'randomSelection',
```

Comment:
`shuffle` would be more appropriate. I would avoid complex, long parameter names like that

Reply:
addressed in9a57bcd8860c2ff4031bed6d103313c4b6c9ba8f